src/interface/graphql/modules/key-result/check-mark/key-result-check-mark.node.ts

```diff
@@ -44,10 +51,16 @@ export class KeyResultCheckMarkGraphQLNode implements GuardedNodeGraphQLInterfac
 
   @Field(() => UserGraphQLNode, {
     complexity: 1,
-    description: 'The user that owns this check mark',
+    description: 'The user that created this check mark',
   })
   public readonly user!: UserGraphQLNode
```

Comment:
Maybe we should change the name of this? To owner? Don't know

Reply:
This is the old field, this means the user who created a checkmark. The best name could be something like `createdBy` or something like this.

As we spoke, this field can be deleted in the future, as we can get this info from logs.